### ISSUE TITLE
delete cluster delete doesn't completely remove cluster context from kube config

### ISSUE THREAD

**Issue by paulbsch** [weaveworks/eksctl#558]
**What happened?**
After running "eksctl delete cluster", the context which was written to the kube config (during create cluster) is delete.  However, it still leaves it set as the "current-context" in the kube config.  (At least in the case when it's the only context in the kube config file.)  Subsequent runs for kubectl will then cause errors as it cannot find the "current-context".

**What you expected to happen?**
That the current-context would be empty in the kube config file.

**How to reproduce it?**
Run "eksctl create cluster" and then "eksctl delete cluster" to delete that same cluster.  (It may require an empty kube config file?)  Then run "kubectl version" or something similar and see that kubectl now complains.

**Anything else we need to know?**
Fedora, downloaded the latest binary rc that was available.

**Versions**
Please paste in the output of these commands:
```
$ eksctl version
[ℹ]  version.Info{BuiltAt:"", GitCommit:"", GitTag:"0.1.20-rc.4"}

$ uname -a
Linux rupert.localdomain 4.16.11-100.fc26.x86_64 #1 SMP Tue May 22 20:02:12 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

$ kubectl version
Client Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.2", GitCommit:"bb9ffb1654d4a729bb4cec18ff088eacc153c239", GitTreeState:"clean", BuildDate:"2018-08-07T23:17:28Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"12", GitVersion:"v1.12.1", GitCommit:"4ed3216f3ec431b140b1d899130a69fc671678f4", GitTreeState:"clean", BuildDate:"2018-10-05T16:36:14Z", GoVersion:"go1.10.4", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by errordeveloper** [comment]
This was fixed in 0.1.21, and today 0.1.22 came out. Please make sure to upgrade :)

**Comment by errordeveloper** [comment]
Duplicate of #527.
